src/openms/source/ANALYSIS/MAPMATCHING/TransformationModelLinear.cpp

```diff
@@ -44,6 +44,9 @@ namespace OpenMS
   {
     params_ = params;
     data_given_ = !data.empty();
+    TransformationModel::DataPoints data_weighted = data;
+    weightData(data_weighted, params); // weight the data
```

Comment:
I dont think this should be the default, lets not apply the transformation and spent all the time doing this when there is no work to be done.